config.json

```diff
@@ -275,6 +275,7 @@
   ],
   "resources": {
     "cart": true,
+    "bulk_discount_rates": true,
```

Comment:
Yes. All pages that have product context. But I'm going to change it to default to `false`